src/platform/packages/shared/kbn-scout/test/scout/.meta/ui/standard.json

```diff
@@ -0,0 +1,132 @@
+{
+  "lastModified": "2025-12-03T19:04:20.842Z",
+  "sha1": "cf987cc228a08cc2083d5d705322a88092e4ba5a",
```

Comment:
Do we need the lastModified and sha1 properties here? Want to make sure it's something we need as it will likely be a cause of merge conflicts.

Reply:
I guess I could get rid of `lastModified`,  but `sha1` is definitely something that must stay. It's needed to determine if the manifest is still accurately representing the scope of the test config it's describing.